charts/mirrorbits/values.yaml

```diff
@@ -5,51 +5,46 @@
 replicaCount:
   mirrorbits: 1
   files: 1
-
 image:
   mirrorbits:
     repository: jenkinsciinfra/mirrorbits
-    tag: v0.1.2
+    tag: 0.1.4
```

Comment:
No `v` in last versions of the docker images: https://hub.docker.com/r/jenkinsciinfra/mirrorbits/tags ✔